src/tutorials/obby_tutorial.md

```diff
@@ -253,7 +255,7 @@ ROTATING_PLATFORM:RotateContinuous(ROTATION_SPEED, false)
 
 Save the script and preview the project. The platform should be rotating and the player should rotate if on top of it.
 
-!!! tip "Use the ++Shift+=++ shortcut to spawn the player in Preview Mode where the creator camera currently is in the scene."
+!!! tip "Use the ++shift+equal++ shortcut to spawn the player in Preview Mode where the scene camera location."
```

Comment:
"where the scene camera location." -> "where the scene camera location current is".